TypeScript - TextArea onChange should provide ChangeEvent event

## Bug Report

### Steps
1. Use TextArea or Form.TextArea.
2. Pass `(e) => setText(e.target.value)` to its onChange.
3. Get a TypeScript error.

https://github.com/Semantic-Org/Semantic-UI-React/blob/9399332b3930a07ff1874bf1d94b7c8a17e30ca6/src/addons/TextArea/TextArea.d.ts#L17

I think `React.ChangeEvent` is suitable for `onChange`, but `React.FormEvent` is used.

Note: `<textarea onChange={(e) => setText(e.target.value)} />` works well.

### Expected Result
Get no TypeScript errors.

### Actual Result
```tsx
error TS2339: Property 'value' does not exist on type 'EventTarget'.

<Form.TextArea value={text} onChange={(e) => setText(e.target.value)} />
                                                              ~~~~~
```

### Version
1.3.0

### Testcase
https://codesandbox.io/s/semantic-ui-react-textarea-typescript-8wyqz

--- a/src/addons/TextArea/TextArea.d.ts
+++ b/src/addons/TextArea/TextArea.d.ts
@@ -14,7 +14,7 @@ export interface StrictTextAreaProps {
    * @param {SyntheticEvent} event - The React SyntheticEvent object
    * @param {object} data - All props and the event value.
    */
-  onChange?: (event: React.FormEvent<HTMLTextAreaElement>, data: TextAreaProps) => void
+  onChange?: (event: React.ChangeEvent<HTMLTextAreaElement>, data: TextAreaProps) => void
 
   /**
    * Called on input.

